libqtile/scratchpad.py

```diff
@@ -184,18 +184,25 @@ def show(self):
         its floating x, y, width and height is set.
         """
         if (not self.visible) or (not self.shown):
+            # SET GEOMETRY
             win = self.window
             screen = win.qtile.current_screen
             # calculate windows floating position and width/height
             # these may differ for screens, and thus always recalculated.
-            win.x = int(screen.dx + self.x * screen.dwidth)
-            win.y = int(screen.dy + self.y * screen.dheight)
-            win.float_x = win.x
-            win.float_y = win.y
-            win.width = int(screen.dwidth * self.width)
-            win.height = int(screen.dheight * self.height)
-
-            # SHOW
+            x = int(screen.dx + self.x * screen.dwidth)
+            y = int(screen.dy + self.y * screen.dheight)
+            width = int(screen.dwidth * self.width)
+            height = int(screen.dheight * self.height)
+            # Set the geometry
+            win.x = x
+            win.y = y
+            win.float_x = x
+            win.float_y = y
+            win.width = width
+            win.height = height
+            # Configure the new geometry
+            win._reconfigure_floating()
```

Comment:
This line here is the only real change of the whole diff, right? Can we remove all the noise?

Reply:
Sure the only required part is `win._reconfigure_floating()`, I will quickly revert the rest.